jbosstools/mars/staging/updates/compositeArtifacts.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <?compositeArtifactRepository version='1.0.0'?>
-<repository name='JBoss Tools 4.3.1.Beta2a Development Milestone Update Site' type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
+<repository name='JBoss Tools 4.3.0.Final Stable Release Update Site (With EGit 4.1.1)' type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
 <properties size='2'>
 <property name='p2.compressed' value='true'/>
-<property name='p2.timestamp' value='1453955143000'/>
+<property name='p2.timestamp' value='1457972311000'/>
 </properties>
-<children size='5'>
-<child location='http://download.jboss.org/jbosstools/targetplatforms/jbosstoolstarget/4.52.0.Beta2-SNAPSHOT/REPO/'/>
-<child location='http://download.jboss.org/jbosstools/mars/staging/updates/core/4.3.1.Beta2a/'/>
-<child location='http://download.jboss.org/jbosstools/targetplatforms/jbtcentraltarget/4.52.0.Beta2-SNAPSHOT/REPO/'/>
-<child location='http://download.jboss.org/jbosstools/mars/staging/updates/central/4.3.1.Beta2a/'/>
-<child location='http://download.jboss.org/jbosstools/mars/staging/updates/integration-stack/'/>
+<children size='6'>
+<child location='http://download.jboss.org/jbosstools/targetplatforms/jbosstoolstarget/4.51.0.Final/REPO/'/>
```

Comment:
We don't have a pre-existing URL pattern for patch fixes like these (which is why I first put it under /stable/updates/JBIDE-#/).

Either we use the one and only /staging/updates/composite*.xml files for such a staged change, or we invent a new one.